pkg/jmxfetch/jmxfetch.go

```diff
@@ -240,6 +240,15 @@ func (j *JMXFetch) Start(manage bool) error {
 			log.Warnf("Java option -XX:MaxRAMPercentage will not take effect since either -Xmx or XX:MaxHeapSize is already present. These options override MaxRAMPercentage.")
 			passOption = false
 		}
+		if !strings.Contains(javaOptions, "java.io.tmpdir") {
+			javaTmpDir := filepath.Join(pkgconfigsetup.Datadog().GetString("run_path"), "jmxfetch")
+			if err := os.Mkdir(javaTmpDir, os.ModeDir|0755); err != nil {
```

Comment:
`Mkdir` returns an error if the directory exists, so this will always fail after the first run. And there is no need to set `ModeDir` explicitly, it is only needed for interpreting file mode returned by `Stat`.

```suggestion
			if err := os.Mkdirs(javaTmpDir, 0755); err != nil {
```